data/authors.yaml

```diff
@@ -69,3 +69,10 @@
 - key: slarocca
   name: Sarah LaRocca
   description: is a Research Scientist on the Demography and Survey Science team at Facebook.
+- key: ajoshi
+  name: Ananya Joshi
+  link: https://csd.cmu.edu/people/graduate-student/ananya-joshi
+  description: is a Ph.D. student in the Computer Science Department and is a member of Delphi.
+- key: cscott
+  name: Chris Scott
+  description: is a Google Fellow and is a member of Delphi.
```

Comment:
weren't we being picky about writing "Google.org Fellow" at one point?

Reply:
Yeah, it should be Google.org.